modules/nodes-nodes-audit-log-basic-viewing.adoc

```diff
@@ -73,40 +70,84 @@ $ oc adm node-logs --role=master --path=kube-apiserver/
+{"kind":"Event","apiVersion":"audit.k8s.io/v1","level":"Metadata","auditID":"cfce8a0b-b5f5-4365-8c9f-79c1227d10f9","stage":"ResponseComplete","requestURI":"/api/v1/namespaces/openshift-kube-scheduler/serviceaccounts/openshift-kube-scheduler-sa","verb":"get","user":{"username":"system:serviceaccount:openshift-kube-scheduler-operator:openshift-kube-scheduler-operator","uid":"2574b041-f3c8-44e6-a057-baef7aa81516","groups":["system:serviceaccounts","system:serviceaccounts:openshift-kube-scheduler-operator","system:authenticated"]},"sourceIPs":["10.128.0.8"],"userAgent":"cluster-kube-scheduler-operator/v0.0.0 (linux/amd64) kubernetes/$Format","objectRef":{"resource":"serviceaccounts","namespace":"openshift-kube-scheduler","name":"openshift-kube-scheduler-sa","apiVersion":"v1"},"responseStatus":{"metadata":{},"code":200},"requestReceivedTimestamp":"2021-03-08T18:06:42.512619Z","stageTimestamp":"2021-03-08T18:06:42.516145Z","annotations":{"authentication.k8s.io/legacy-token":"system:serviceaccount:openshift-kube-scheduler-operator:openshift-kube-scheduler-operator","authorization.k8s.io/decision":"allow","authorization.k8s.io/reason":"RBAC: allowed by ClusterRoleBinding \"system:openshift:operator:cluster-kube-scheduler-operator\" of ClusterRole \"cluster-admin\" to ServiceAccount \"openshift-kube-scheduler-operator/openshift-kube-scheduler-operator\""}}
+----
+
+* View the OpenShift OAuth API server logs:
+
+.. If necessary, get the node name of the log you want to view:
++
+[source,terminal]
+----
+$ oc adm node-logs --role=master --path=oauth-apiserver/
+----
++
+.Example output
+[source,terminal]
+----
+ci-ln-m0wpfjb-f76d1-vnb5x-master-0 audit-2021-03-09T13-06-26.128.log
+ci-ln-m0wpfjb-f76d1-vnb5x-master-0 audit.log
+ci-ln-m0wpfjb-f76d1-vnb5x-master-1 audit-2021-03-09T18-23-21.619.log
+ci-ln-m0wpfjb-f76d1-vnb5x-master-1 audit.log
+ci-ln-m0wpfjb-f76d1-vnb5x-master-2 audit-2021-03-09T17-36-06.510.log
+ci-ln-m0wpfjb-f76d1-vnb5x-master-2 audit.log
+----
+
+.. View the OpenShift OAuth API server log for a specific master node and timestamp or view all the logs for that master:
++
+[source,terminal]
+----
+$ oc adm node-logs <node_name> --path=oauth-apiserver/<log_name>
+----
++
+For example:
++
+[source,terminal]
+----
+$ oc adm node-logs ci-ln-m0wpfjb-f76d1-vnb5x-master-0 --path=oauth-apiserver/audit-2021-03-09T13-06-26.128.log
+----
++
+[source,terminal]
+----
+$ oc adm node-logs ci-ln-m0wpfjb-f76d1-vnb5x-master-2 --path=oauth-apiserver/audit.log
+----
++
+The output appears similar to the following:
```

Comment:
You could potentially remove this line because the `.Example output` title conveys the same message.